.github/workflows/test.yml

```diff
@@ -20,34 +20,25 @@ jobs:
           profile: minimal
           toolchain: ${{ matrix.rust }}
           override: true
-      - name: test/debug
+      - name: check/benches
         uses: actions-rs/cargo@v1
         with:
-          command: test
-      - name: test/release
+          command: check
+          args: --benches
+      - name: check
```

Comment:
You could  optimise even more by having a single `check--all-features --all-targets` rather than compile for `check` twice. Then you can remove the check/benches job

Reply:
Nice! I didn't know those flags would cover benches.